deploy/helm/kuberhealthy/templates/deployment.yaml

```diff
@@ -31,6 +31,16 @@ spec:
       automountServiceAccountToken: true
       containers:
       - image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+        volumeMounts:
+        - name: config-volume
+          mountPath: /etc/config/
+      volumes:
+       - name: config-volume
+         configMap:
+        # Provide the name of the ConfigMap containing the files you want
+        # to add to the container
+           name: special-config
```

Comment:
the name of the configmap should not be `special-config`.  Let's make it simply `kuberhealthy`.  The type will be obvious.

Reply:
Changes made with 6bf8b54